app/Http/Controllers/ArticleController.php

```diff
@@ -15,16 +17,18 @@ class ArticleController extends Controller
      *
      * @return \Illuminate\Http\Response
      */
-    public function index()
+    public function index(Request $request)
```

Comment:
```suggestion
    public function index()
```

一応、使ってないから引数requestいらんかな

Reply:
削除済。